CMakeLists.txt

```diff
@@ -2050,6 +2051,11 @@ elseif(APPLE)
     set(HAVE_SDL_FILE TRUE)
   endif()
 
+  if(IOS OR TVOS OR MACOSX OR DARWIN)
```

Comment:
You're not adding sources for android/unix/windows, only apple platforms.

Reply:
android / unix is automatic as it;s .c extension.
it should be ok now ?

Reply:
You're right. My ba